pyrk/neutronics.py

```diff
@@ -71,7 +98,7 @@ def init_rho_ext(self, rho_ext):
             rho_ext = ReactivityInsertion(self._timer)
         return rho_ext
 
-    def dpdt(self, t_idx, components, power, zetas):
+    def dpdt(self, t_idx, components, power, zetas, zeta_refs=[]):
```

Comment:
please add a definition in the docstring for "zeta_refs".  Does this really need to be passed into the function?